packages/react-composites/src/composites/CallWithChatComposite/adapter/AzureCommunicationCallWithChatAdapter.ts

```diff
@@ -853,19 +863,25 @@ export type AzureCommunicationCallWithChatAdapterFromClientArgs = {
  *
  * @public
  */
-export const createAzureCommunicationCallWithChatAdapterFromClients = async ({
-  callClient,
-  callAgent,
-  callLocator,
-  chatClient,
-  chatThreadClient
-}: AzureCommunicationCallWithChatAdapterFromClientArgs): Promise<CallWithChatAdapter> => {
+export async function createAzureCommunicationCallWithChatAdapterFromClients(
```

Comment:
any reason why this change from const to function?

Reply:
The underlying functionality is the same, we just need to change it from an arrow function to just this normal function declaration cause it is the only way to include conditional compile flags in the arguments. This is a known limitation, we had to make this change in the ChatAdapter as well